src/core/logical_layers/renderers/MCDARenderer/MCDARenderer.tsx

```
@@ -0,0 +1,32 @@
+import type { ApplicationMap } from '~components/ConnectedMap/ConnectedMap';
+import type { LogicalLayerState } from '~core/logical_layers/types/logicalLayer';
+import type { MCDALayerStyle } from '../stylesConfigs/mcda/types';
+import type { MultivariateLayerStyle } from '../stylesConfigs/multivariate/multivariateStyle';
+
+export class MCDARenderer extends MultivariateRenderer {
+  protected getSourcePrefix(): string {
+    return 'mcda-source-';
+  }
+
+  protected getClickableLayerId(): string {
+    return 'mcda-layer-' + this.id;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)